### DISCUSSION TITLE
Localization Support in Bertin.js

### DISCUSSION THREAD

**Discussion by arky** [neocarto/bertin#107] [Ideas]
I have briefly created a bug for adding built-in support for localization. 

https://github.com/neocarto/bertin/issues/105

Support for localization for D3 ecosystem has been sparse.  Maps are always monolingual,  we often end up creating multiple maps for each language which ends up as maintenance nightmare in the long term. 
 
Let's get this discussion started.

**Comment by neocarto** [comment]
Ok. I see. But not so easy to do it right. If you have any specific suggestions, let me know.

**Reply by arky** [reply]
Indeed, the first step is identify a framework: [formatjs](http://formatjs.io/), [ployglot](http://airbnb.io/polyglot.js/), [i18next](https://www.i18next.com/) which is most flexible to use with D3.js. Unfortunately, i18n has not be high on D3 development goals.
